library/certificate_request.py

```diff
@@ -28,6 +28,11 @@
         certificate. Also can provide the default value for
         common_name.
     required: false
+  key_size:
```

Comment:
This should also be documented in the `README.md` for the role parameters - this is just for the module parameters.

Reply:
I was planning to add the README from our specs as the start point. I just didn't add yet because we are actually missing most of what's there.